mmdet/models/detectors/rpn.py

```diff
@@ -106,10 +106,17 @@ def simple_test(self, img, img_metas, rescale=False):
             list[np.ndarray]: proposals
         """
         x = self.extract_feat(img)
+        # get origin input shape to onnx dynamic input shape
+        import torch
```

Comment:
Do not import torch in the test function because it will import torch every test iteration and will slow down the testing speed.